ksp_plugin_adapter/window_renderer.cs

```diff
@@ -6,26 +6,26 @@ namespace ksp_plugin_adapter {
 // A helper class for scaling the UI.  Unlike the WindowRenderer below, it can
 // be used for elements that are not windows.
 internal class ScalingRenderer {
-  protected ScalingRenderer() {
+  static ScalingRenderer() {
     // All dimensions are expressed in "units".  By default a unit is 25 pixels
     // but it can scale up or down based on the KSP UI scale.
     scale_ = GameSettings.UI_SCALE * GameSettings.UI_SCALE_APPS;
```

Comment:
There is no guarantee that UI_SCALE and UI_SCALE_APPS will have been set when the static constructor runs.